### PR TITLE
vbindiff: update to 3.0_beta5, use HTTPS link

### PR DESCRIPTION
**Pull request by chrstphrchvz**
#### Description
- use HTTPS link for homepage/fetch
- update to 3.0_beta5:

>  VBinDiff 3.0 beta 5 — 10 Sep 2017
VBinDiff development now uses Git, and the source is available on [GitHub](https://github.com/madsen/vbindiff)
The Windows version is now built with Visual Studio 2015
Merged [GitHub #3](https://github.com/madsen/vbindiff/pull/3): Fix crash when comparing files (by [nfllab](https://github.com/nfllab))
Merged [GitHub #6](https://github.com/madsen/vbindiff/pull/6): Build on systems with split ncurses/tinfo libraries (by [laomaiweng](https://github.com/laomaiweng))

###### Type(s)

- [x] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.13.4 17E202
Xcode 9.3.1 9E501

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] ~~referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?~~

- [x] checked your Portfile with `port lint`?
- [ ] ~~tried existing tests with `sudo port test`?~~ [No tests for vbindiff.]
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: sysutils/vbindiff/Portfile [hunk 1]
```diff
@@ -14,13 +13,14 @@ long_description    VBinDiff (Visual Binary Diff) displays files in \
...
+homepage        https://www.cjmweb.net/vbindiff/
 platforms       darwin
 license         GPL-2+
 master_sites    ${homepage}
 
-checksums       md5 dbda80ef580e1a0975ef50b9aaa5210e \
-                sha1 0e8c63debafe3d5bb8741e0254c95457d78c7f2d \
-                rmd160 156fa68891d4aa9c0280177c70e1a3d53b912d13
+checksums       md5     bbde86b790d6a912e3ba11d6cd8b1a2a \
+                sha1    cbd4651f604ad18cf8d9122b3c2719d0173218f7 \
```
Comment (pmetzger) [inline sysutils/vbindiff/Portfile]: Please remove the "md5" and "sha1" checksums, we no longer use those. You should also add a "size" field at the end while you're at it.

**Comment by pmetzger** [comment]
The maintainer is apparently "yahoo.com:wickedguitar6", which, given how few hackers still use yahoo addresses and the fact that we have no github handle for this person, makes me wonder if the maintainer still uses that email. Please get in touch with them and alert them to this pull request and ask them for their github handle.

**Comment by chrstphrchvz** [comment]
@pmetzger, I was wondering the same thing, as the maintainer hasn't been active on macports for several years possibly. I will send them an email if necessary.

@gamanakis, by chance do you happen to be wickedguitar6? Sorry for bothering you if you aren't--there was a Launchpad.net user with the same combination of handles.

**Comment by mf2k** [comment]
I opened a port abandoned ticket for vbindiff. 

(https://trac.macports.org/ticket/56555)

**Comment by chrstphrchvz** [comment]
I guess the "type: update" label only gets applied automatically if the pull is named "portfile: update to &hellip;" on the first try.

**Comment by pmetzger** [comment]
Merged! Thank you, @chrstphrchvz. I'm presuming, btw, that this port has been abandoned, so if you want to assume maintainership, submit a pull request after the main ticket times out.

**Pull request merged**